### PR TITLE
Jeffboudier post

### PR DESCRIPTION
**Pull request by jeffboudier**
Blog post for publication on Tuesday 2/21 at 9:30am PST

### CODE REVIEW

**Review by philschmid — APPROVED** [review]
Awesome Blog! 🔥

**Comment by mrm8488** [comment]
> Blog post for publication on Tuesday 2/21 at 9:30am PST

Love it!

File: aws-partnership.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Hugging Face and AWS partner to make AI more accessible"
```
Comment (julien-c) [inline aws-partnership.md:2]:
```suggestion
title: "Hugging Face and AWS partner to make AI more open" 
```

File: aws-partnership.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Hugging Face and AWS partner to make AI more accessible" 
+thumbnail: /blog/assets/131_aws-partnership/aws-partnership-thumbnail.png
+authors:
+- user: jeffboudier
+- user: philschmid
+- user: juliensimon
+---
+
+# Hugging Face and AWS partner to make AI more accessible
```
Comment (julien-c) [inline aws-partnership.md:10]:
```suggestion
# Hugging Face and AWS partner to make AI more open
```

File: _blog.yml [hunk 3]
```diff
@@ -1796,4 +1808,15 @@
     - guide
     - nlp
     - cv
-    - multimodal
\ No newline at end of file
+    - multimodal
+
+- local: aws-partnership
+  title: "Hugging Face and AWS partner to make AI more accessible"
```
Comment (julien-c) [inline _blog.yml:1814]:
```suggestion
  title: "Hugging Face and AWS partner to make AI more open"
```

**Review by julien-c — COMMENTED** [review]
Tweak proposal (will need to update the thumbnail if this proposal is accepted)

**Pull request merged**